### DISCUSSION TITLE
How to override prisma to run in cypress e2e tests

### DISCUSSION THREAD

**Discussion by Anakin100100** [prisma/prisma#19724] [Q&A]
I'd like to be able to use prisma inside cypress e2e tests to validate the db state and reproduce some manual onboarding tasks that haven't been coded into the app yet. Unfortunately I'm getting an error "PrismaClient is unable to be run in the browser" regardless of where I try to initialize the client. How can I override this behavior of the client?

**Comment by ludralph** [comment]
Hi @Anakin100100 👋 

Can you please provide more information on how you are using Prisma with Cypress? Code examples will help here as well. Also, be aware that you cannot use Prisma within the context of the browser but you can leverage [Cypress Task](https://docs.cypress.io/api/commands/task) to interface with Prisma and perform database operations.

**Reply by ludralph** [reply]
Hey @Anakin100100 ,

It would be really helpful if you could provide the additional information that we had asked about last time. If your problem is resolved by now or somehow doesn't feel relevant any more, you can simply close the Discussion as well.

**Reply by nurul3101** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have a similar problem 🙏
